examples/siro_sandbox/app_states/app_state_fetch.py

```diff
@@ -112,14 +112,14 @@ def _update_grasping_and_set_act_hints(self):
                 (
                     candidate_walk_dir,
                     candidate_distance_multiplier,
-                ) = self._nav_helper._get_humanoid_walk_dir_from_remote_gui_input(
+                ) = self._nav_helper.get_humanoid_walk_hints_from_remote_gui_input(
                     visualize_path=False
                 )
             else:
                 (
                     candidate_walk_dir,
                     candidate_distance_multiplier,
-                ) = self._nav_helper._get_humanoid_walk_dir_from_ray_cast(
+                ) = self._nav_helper.get_humanoid_walk_dir_from_ray_cast(
```

Comment:
why still "walk_dir"? Make sure to test these codepaths given we don't have CI hooked up yet!

Reply:
done